plugins/modules/datahub_template_info.py

```diff
@@ -42,6 +42,13 @@
     required: False
     aliases:
       - template
+  content:
```

Comment:
not to be a sophist, but I think this would be better as `return_content` as that is an instruction. I get that _info modules are always only to return things, but it feels more correct to me to be explicit?

Reply:
+1 -- in addition, using `return_content` would be consistent with `ansible.builtin.uri`: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/uri_module.html#parameter-return_content